app/components/Login.react.js

```diff
@@ -52,25 +54,61 @@ class Login extends Component {
         this.buildOauthUrl = this.buildOauthUrl.bind(this);
         this.oauthPopUp = this.oauthPopUp.bind(this);
         this.logIn = this.logIn.bind(this);
-        this.checkAnyUserAccountIsSaved = this.checkAnyUserAccountIsSaved.bind(this);
     }
 
-    componentWillReceiveProps(nextProps) {
-        this.checkAnyUserAccountIsSaved().then(isSaved => {
-            if (isSaved) {
-                window.location.assign(connectorUrl);
-            }
-        });
+    componentDidMount() {
+        const {serverType} = this.state;
+        const interval = setInterval(() => {
+              usernameLogged = cookie.load('db-connector-user');
```

Comment:
could we add a comment here explaining why we're only looking at cookies and not the ipc state? in other words, why this block of code doesn't pertain to the electron environment